distributed/client.py

```diff
@@ -2767,6 +2767,17 @@ def scheduler_info(self, **kwargs):
         self.sync(self._update_scheduler_info)
         return self._scheduler_identity
 
+    def to_scheduler_file(self, scheduler_file):
```

Comment:
Thoughts on calling this `write_scheduler_file`?  The prefix `to_` is often used in projects like pandas and xarray when we convert the data in the object to a new format.  That isn't quite the case here.